e2e/node/acp-v3.test.ts

```diff
@@ -0,0 +1,271 @@
+    );
+    selfWriteNoReadPolicy = acp.addAllOfRuleUrl(
+      selfWriteNoReadPolicy,
+      selfRule
+    );
+    selfWriteNoReadPolicy = acp.setAllowModes(selfWriteNoReadPolicy, {
+      read: false,
+      append: true,
+      write: true,
+    });
+    selfWriteNoReadPolicy = acp.setDenyModes(selfWriteNoReadPolicy, {
+      read: true,
+      append: false,
+      write: false,
+    });
+
+    let policyResource = createSolidDataset();
+    policyResource = setThing(policyResource, publicRule);
+    policyResource = setThing(policyResource, publicReadPolicy);
+    policyResource = setThing(policyResource, selfRule);
+    policyResource = setThing(policyResource, selfWriteNoReadPolicy);
+
+    return saveSolidDatasetAt(policyResourceUrl, policyResource, options);
+  }
+
+  async function applyPolicyToPolicyResource(
+    resourceUrl: UrlString,
+    policyUrl: UrlString,
+    session: Session
+  ) {
+    const resourceWithAcr = await acp.getSolidDatasetWithAcr(resourceUrl, {
+      fetch: session.fetch,
+    });
+    if (!acp.hasAccessibleAcr(resourceWithAcr)) {
+      throw new Error(
+        `The test Resource at [${getSourceUrl(
+          resourceWithAcr
+        )}] does not appear to have a readable Access Control Resource. Please check the Pod setup.`
+      );
+    }
+    const changedResourceWithAcr = acp.addPolicyUrl(
+      resourceWithAcr,
+      policyUrl
+    );
+    return acp.saveAcrFor(changedResourceWithAcr, options);
+  }
+
+  it("can deny Read access", async () => {
+    // Create a Resource containing Access Policies and Rules:
+    await initialisePolicyResource(sessionResource, session);
+
+    // Verify that we can fetch the Resource before Denying Read access:
+    await expect(
+      getSolidDataset(sessionResource, options)
+    ).resolves.not.toBeNull();
+
+    // In the Resource's Access Control Resource, apply the Policy
+    // that just so happens to be defined in the Resource itself,
+    // and that denies Read access to the current user:
+    await applyPolicyToPolicyResource(
+      sessionResource,
+      sessionResource + "#policy-selfWriteNoRead",
+      session
+    );
+
+    // Verify that indeed, the current user can no longer read it:
+    await expect(
+      getSolidDataset(sessionResource, options)
+    ).rejects.toThrow(
+      // Forbidden:
+      // @ts-ignore-next
+      expect.objectContaining({ statusCode: 403 }) as FetchError
+    );
+
+    // Clean up:
+    await deleteSolidDataset(sessionResource, options);
+  });
+
+  it("can allow public Read access", async () => {
+    // Create a Resource containing Access Policies and Rules:
+    await initialisePolicyResource(sessionResource, session);
+
+    // Verify that we cannot fetch the Resource before adding public Read access
+    // when not logged in (i.e. not passing the session's fetch):
+    await expect(getSolidDataset(sessionResource)).rejects.toThrow(
+      // Unauthorised:
+      // @ts-ignore-next
```

Comment:
ditto

Reply:
I don't think it's needed, removed in https://github.com/inrupt/solid-client-js/pull/1426/commits/b63d01405ca610982d8680df69e4b6e3d8341386.